### DISCUSSION TITLE
👽🤠🤖 Who is using Netmaker?

### DISCUSSION THREAD

**Discussion by afeiszli** [gravitl/netmaker#1615] [Polls]
### Discussed in https://github.com/gravitl/netmaker/discussions/859
  
<div type='discussions-op-text'>
  
<sup>Originally posted by **afeiszli** March  4, 2022</sup>
[👋](https://emojipedia.org/waving-hand/) Hey Netmaker Users,
  
If your company is using Netmaker in production, please comment below so we can mention you in the README / website.  
  
Here's what your comment should look like:  
  
Company: [add name here]
Permission to use logo in README/website: [Y/N]
Project description: [Short description of how you are using Netmaker]
  
Thank you!</div>

**Comment by stianlaa** [comment]
Hi Netmaker, thanks for a great product thus far, looking forward to see where you're taking this :tada: 

**Company:** Nordic Unmanned
**Permission to use logo in README/website:** Yes
**Project description:** We are using Netmaker to facilitate communication between various types of drone and drone operations systems in a safe and convenient manner. It gives us the many benefits of wireguard, and hides much of the complexity of the underlying network.

Here are a couple of Logo's for your convenience:
![nu_logo_rgb_orangeonblack](https://user-images.githubusercontent.com/16704943/162738285-9b8d4aaa-6eec-4392-8d83-7747ea94feeb.png)
![nu_logo_rgb_colour](https://user-images.githubusercontent.com/16704943/162738289-59e628a5-3666-46d0-972f-9836f1f97225.png)
